libraries/DataFlash/LogStructure.h

```diff
@@ -873,6 +873,18 @@ struct PACKED log_Proximity {
     float closest_dist;
 };
 
+struct PACKED log_SRTL {
+    LOG_PACKET_HEADER;
+    uint64_t time_us;
+    uint8_t active;
+    uint16_t num_points;
```

Comment:
Yes, Peter has brought  up as well that we should be using uint16_t for all our indexing into the arrays instead of int16_t.  I plan to change this hopefully tomorrow.  It's quite a fundamental change though so I suspect I will break things so I want to do it last.